{{ cookiecutter.repo_name }}/Makefile

```diff
@@ -16,13 +16,13 @@ include .cookiecutter/config
 
 # Ensure directory to track and log setup state exists
 $(shell mkdir -p .cookiecutter/state)
-$(shell touch .cookiecutter/state/conda-create.log)
+$(shell touch .cookiecutter/state/python-env-create.log)
 
 .PHONY: install
-## Install a project: remove existing conda env (if exists); create conda env; install local package; setup git hooks
-install: conda-remove .cookiecutter/state/conda-create .cookiecutter/state/setup-git
-	@direnv reload  # Now the conda env exists, reload to activate it
-
+## Install the project: setup environment, install dependencies, and configure git hooks
+install:
+    $(MAKE) python-env-remove .cookiecutter/state/python-env-create .cookiecutter/state/setup-git
```

Comment:
Receiving the following error when trying to test the branch, just after running `make install`:

```
Makefile:24: *** missing separator.  Stop.
```

Reply:
I am not too familiar with makefiles, but think something about how the dependencies are ordered isn't right?